avsomr/avsomrnetrecognizer.cpp

```diff
@@ -0,0 +1,210 @@
+#include "avsomrnetrecognizer.h"
+
+#include <QFile>
+#include <QFileInfo>
+#include <QNetworkAccessManager>
+#include <QHttpMultiPart>
+#include <QNetworkRequest>
+#include <QNetworkReply>
+#include <QEventLoop>
+#include <QTimer>
+#include <QJsonDocument>
+#include <QJsonObject>
```

Comment:
should be covered by all.h

Reply:
I will fix this if it is accepted, but then the developer does not show what is used in the class and my IDE does not highlight types and auto-complite does not work. Need to include "all.h"?